cobbler/items/item.py

```diff
@@ -467,39 +458,91 @@ def _resolve(self, property_name: str) -> Any:
                 f'{type(self)} "{self.name}" does not have property "{property_name}"'
```

Comment:
No those are different cases. One case refers to the object itself having the attribute, and the other refers to the case that the parent item or settings don't have the attribute. Lastly, the error highlighted in this snippet is raising an exception in case the method can't return correct data which this method must do in all cases.

Reply:
The surrounding code is
```python
        if not hasattr(self, attribute):
            raise AttributeError(
                f'{type(self)} "{self.name}" does not have property "{property_name}"'
            )

        return getattr(self, attribute), settings_name
```
Both `hasattr` and `getattr` are called with the same arguments,  which is fine if we do something with the result of `hasattr`. But here we just throw the same exception that `getattr` would throw as well.

Reply:
Okay I was having the opinion that enriching the item type and item name is useful for the user but in general this error message shouldn't be thrown anyway and as such this should be fine. I did a rebase and merge on your PR.